content/common/makefile/linux.mk

```diff
@@ -0,0 +1,12 @@
+MAKEFILE_DIR = $(dir $(lastword $(MAKEFILE_LIST)))
+include $(MAKEFILE_DIR)/defs.mk
+
+SRCS := $(wildcard *.c)
```

Comment:
Be consistent about `=` and `:=`. At this point it doesn't make sense to use both of them. See [the difference](https://stackoverflow.com/questions/448910/what-is-the-difference-between-the-gnu-makefile-variable-assignments-a). It's confusing for someone who checks it and then doesn't know why one or the other is being used.